src/app.md

```diff
@@ -189,6 +190,85 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-tar-decompress-remove-path-prefix]: #ex-tar-decompress-remove-path-prefix
+<a name="ex-tar-decompress-remove-path-prefix"></a>
+## Decompress a tarball while removing a prefix from the paths
+
+[![flate2-badge]][flate2] [![tar-badge]][tar] [![cat-compression-badge]][cat-compression]
+
+Decompresses the tarball by iterating over the entries and unpacking them after
+removing a specified path prefix.
+
+Create a [`tar::Archive`] from a [`flate2::read::GzDecoder::new`], iterate over
```

Comment:
Please note that there are already less advanced examples for these crates showing the decompression. So we might like to focus on the stripping prefix part.